autoware_launch/config/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/obstacle_stop_planner.param.yaml

```diff
@@ -26,7 +26,7 @@
         pedestrian_lateral_margin: 0.0       # margin of pedestrian footprint [m]
         unknown_lateral_margin: 0.0          # margin of unknown footprint [m]
         step_length: 1.0                     # step length for pointcloud search range [m]
-        extend_distance: 0.0                 # extend trajectory to consider after goal obstacle in the extend_distance [m]
+        enable_stop_behind_goal_for_obstacle: True # enable extend trajectory after goal lane for obstacle detection
```

Comment:
I think this should be true not True.
Did you test planning simulator with this parameter？

Reply:
Yes i tested it, there is no problem for me. Moreover at this param file, other boolean params written by this way. (False, True)

https://github.com/autowarefoundation/autoware.universe/blob/main/planning/obstacle_stop_planner/config/obstacle_stop_planner.param.yaml